src/main.js

```diff
@@ -90,6 +95,27 @@ export class Draggable {
             unbind(document, "mouseup", this._mouseup);
             this._releaseHandler(e);
         };
+
+        this._pointerdown = (e) => {
+            if (e.isPrimary) {
+                e.target.style.touchAction = "none";
+                e.target.setPointerCapture(e.pointerId);
+                this._pressHandler(e);
+            }
+        };
+
+        this._pointermove = (e) => {
+            if (e.isPrimary) {
+                this._dragHandler(e);
+            }
+        };
+
+        this._pointerup = (e) => {
+            if (e.isPrimary) {
+                this._releaseHandler(e);
+                e.target.style.touchAction = "auto";
```

Comment:
Maybe we should reset the `touchAction` to its original value (from pointerdown). What do you think?

Reply:
Nice catch, it could've been anything.